linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/bash/shared.sh

```diff
@@ -8,9 +8,9 @@ config_file="/etc/ntp.conf"
 
 
 # Set maxpoll values to var_time_service_set_maxpoll
-sed -i "s/^\(server.*maxpoll\) [0-9][0-9]*\(.*\)$/\1 $var_time_service_set_maxpoll \2/" "$config_file"
+sed -i "s/^\(\(server\|pool\).*maxpoll\) [0-9][0-9]*\(.*\)$/\1 $var_time_service_set_maxpoll \3/" "$config_file"
```

Comment:
@yuumasato , is this `\3` back-reference necessary in any situation I can't see? Or could be removed?

Reply:
The last back reference is necessary to keep any `server` or `pool` otions defined after `maxpool`.

It increased from `\2` to `\3` because I did not manage to get non-capturing groups to work with `sed`.
And it seems it is not supported: https://stackoverflow.com/a/36546377

Reply:
Ok. Thanks